src/routing/routing.component.test.tsx

```diff
@@ -94,6 +95,36 @@ describe('Routing component', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('does not render a route for a plugin when site is under maintenance', () => {
+    const maintenancePageClasses = {
+      root: 'root-class',
+      container: 'container-class',
+      titleText: 'titleText-class',
+    };
+
+    state.scigateway.maintenance = { show: true, message: 'test' };
+    state.scigateway.plugins = [
+      {
+        section: 'test section',
+        link: '/test_link',
+        plugin: 'test_plugin_name',
+        displayName: 'Test Plugin',
+        order: 1,
+      },
+    ];
+    const wrapper = mount(
+      <Provider store={mockStore(state)}>
+        <MemoryRouter
+          initialEntries={[{ key: 'testKey', pathname: '/test_link' }]}
+        >
+          <MaintenancePage classes={maintenancePageClasses} />
```

Comment:
Should this be `<Routing ... />` like in the above test in order to test the logic in `routing.component.tsx`? We already test the `<MaintenancePage />` component renders correctly in `maintenancePage.component` right?

Reply:
It was erroring when I used `<Routing ... />` so I thought that `<MaintenancePage />`. However, mocking the `MaintenancePage` component as per your suggestion in the other comment seems to work.